functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/BuildConfigurator.java

```diff
@@ -183,11 +189,23 @@ public BuildConfigurator excludeProjects(String... projects) {
         return this;
     }
 
-    public BuildConfigurator useMavenVersion(String mavenVersion){
+    public BuildConfigurator useMavenVersion(String mavenVersion) {
         this.mavenVersion = mavenVersion;
         return this;
     }
 
+    public BuildConfigurator useSurefireVersion(String version) {
+        File pomFile = new File(projectBuilder.getRoot().toAbsolutePath().toString()+ File.separator + "pom.xml");
+        try {
+            Model model = new MavenXpp3Reader().read(new FileInputStream(pomFile));
+            model.getProperties().setProperty("version.surefire.plugin", version);
```

Comment:
What if my repository is has defined surefire configuration like 

```xml
<plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-surefire-plugin</artifactId>
        <version>2.19.1</version>
  </plugin>
```
I believe it will take defined in surefire plugin not in properties.

Reply:
Yes, exactly... It should have been set there.

Reply:
In other words, check if the pom contains the plugin definition, if yes then set the version there. If the plugin is not specified, then add the whole plugin definition there.

Reply:
I have checked for this case and it works when set in properties. And this code is only for the test bed, so I don't expect the user to change the structure of the pom.

Reply:
Setting it inside the plugin configuration section of the pom requires extensive parsing of the pom, already implemented as part of this [PR](https://github.com/arquillian/smart-testing/pull/223).

If you review that and it looks fine, I can follow the same here as well.

Reply:
Yeah, you are right - sorry for confusions. I was applying//trying it on a wrong project. It works with the dogfooding repo.